pkg/resources/device_selectors.go

```diff
@@ -24,6 +24,26 @@ func (s *vendorSelector) Filter(inDevices []types.PciNetDevice) []types.PciNetDe
 	return filteredList
 }
 
+type deviceSelector struct {
```

Comment:
name is kinda generic as everything here is a device selector

perhaps: `deviceIDSelector` ?